Makefile

```diff
@@ -6,6 +6,7 @@ BUILD_HASH_SHORT = $(shell git rev-parse --short HEAD)
 LDFLAGS += -X "github.com/mattermost/mattermost-marketplace/internal/api.buildTag=$(BUILD_TAG)"
 LDFLAGS += -X "github.com/mattermost/mattermost-marketplace/internal/api.buildHash=$(BUILD_HASH)"
 LDFLAGS += -X "github.com/mattermost/mattermost-marketplace/internal/api.buildHashShort=$(BUILD_HASH_SHORT)"
+SLS_STAGE ?= "dev"
```

Comment:
This change isn't strictly necessary for this PR, since I invoke `sls` manually from the CirlceCI config (too much nonsense about preserving environment variables, sudo, CircleCI, etc.), but it's useful for deploying by hand.